dist/windows/config.nsh

```diff
@@ -47,29 +36,16 @@
 ;==============================================================================
 
 !ifndef QBT_INSTALLER_WINDOWNAME | QBT_INSTALLER_FILENAME
-  !ifndef QBT_IS_X64
-    ; The name of the installer
-    !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION}"
-
-    ; The file to write
-    !define QBT_INSTALLER_FILENAME "${QBT_VERSION}"
-  !else ; QBT_IS_X64
-    !ifndef QBT_USES_QT6
-      ; The name of the installer
-      !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION} x64"
-
-      ; The file to write
-      !define QBT_INSTALLER_FILENAME "${QBT_VERSION}_x64"
-    !else ; QBT_USES_QT6
-      ; The name of the installer
-      !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION} (qt6) x64"
-
-      ; The file to write
-      !define QBT_INSTALLER_FILENAME "${QBT_VERSION}_qt6_x64"
-    !endif ; QBT_USES_QT6
-  !endif ; QBT_IS_X64
+  ; The name of the installer
+  !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION}"
```

Comment:
I would like the default value to be the equivalent of the old `QBT_IS_X64` code path.
The non `QBT_IS_X64` code path was for the now deprecated 32-bit version.

Reply:
> I would like the default value to be the equivalent of the old `QBT_IS_X64` code path. The non `QBT_IS_X64` code path was for the now deprecated 32-bit version.

What do you mean? x64 suffix?

Reply:
Does it make sense to mention x64 and/or Qt6 if there are no other options now, only x64 Qt6?

Reply:
>What do you mean? x64 suffix?
```nsis
; The name of the installer
!define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION} x64"
; The file to write
!define QBT_INSTALLER_FILENAME "${QBT_VERSION}_x64"
```

>Does it make sense to mention x64 and/or Qt6 if there are no other options now, only x64 Qt6?

x64 suffix yes. IMO, we should be explicit for the arch. Plus, the update-check code won't need yet another tweak.

The qt6 is not needed anymore from the update-check code IIRC.

Reply:
> > What do you mean? x64 suffix?
> 
> ```nsis
> ; The name of the installer
> !define QBT_INSTALLER_WINDOWNAME "${QBT_VERSION} x64"
> ; The file to write
> !define QBT_INSTALLER_FILENAME "${QBT_VERSION}_x64"
> ```

Done.